### DISCUSSION TITLE
How to enable GitHub co-pilot using GitHub education?

### DISCUSSION THREAD

**Discussion by carbinecodes** [community/community#64044] [GitHub Education] [answered]
### Hi everyone,

I was looking for github copilot what is the process for claim it using education? do I need to enable github pro before?

**Comment by arjunkumarbose** [comment, marked as answer]
Check this document, it will answer all the questions. Along with yes, you need to claim the GitHub education pro first, then it will actually enables copilot as well.
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

**Reply by carbinecodes** [reply]
Thanks a lot!
